x/emissions/module.go

```diff
@@ -135,7 +135,7 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 	types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
 	types.RegisterQueryServer(cfg.QueryServer(), am.keeper)
 	m := keeper.NewMigrator(am.keeper, am.legacySubspace)
-	if err := cfg.RegisterMigration(types.ModuleName, 5, m.Migrate5to6); err != nil {
+	if err := cfg.RegisterMigration(types.ModuleName, 6, m.Migrate6to7); err != nil {
```

Comment:
@kingpinXD what is the best way to run upgrade tests now?

Reply:
ok thanks i will merge it, we will run complete upgrade tests anyways before release
PR is basically the same as last time we did it, i dont think anything is missing, we just need migration which i added